server/chat-plugins/uno.ts

```diff
@@ -763,7 +763,7 @@ export const commands: ChatCommands = {
 			const game = room.getGame(UnoGame);
 			if (!game) return this.errorReply("There is no UNO game going on in this room right now.");
 			if (!this.canTalk()) return false;
-			if (!game.joinGame(user)) return this.errorReply("Unable to join the game.");
+			if (!game.joinGame(user)) return this.errorReply("You have already joined the game of UNO.");
```

Comment:
Is this the only case `joinGame` can fail?

Reply:
I don't believe so, no.